manager_test_helper.go

```diff
@@ -354,13 +374,15 @@ func TestHelperCreateGetDelete(s Manager) func(t *testing.T) {
 			found := map[string]int{}
 			for _, got := range pols {
 				for _, expect := range TestManagerPolicies {
-					if got.GetID() == expect.GetID() {
```

Comment:
Could you please replace this with

```
 			for _, got := range pols {
 				for _, expect := range TestManagerPolicies {
					if got.GetID() == expect.GetID() {
                        assert.True(reflect.DeepEquals(expect, got))
 						found[got.GetID()]++
 					}
 				}
 			}
```

Reply:
@adhicoder please do this

Reply:
Still failing. Not sure why though.

Reply:
That's ok, and was also the whole point of this PR (creating a sane failing test to find the bug you described). However, the output is still not conclusive enough (https://travis-ci.org/ory/ladon/builds/338007648#L605). Let's change `assert.True(t, reflect.DeepEquals(expect, got))` to `assert.ObjectsAreEqualValues(t, expect, got)` which will print a better error message